spec/models/doorkeeper/access_token_spec.rb

```diff
@@ -19,8 +19,6 @@ def self.generate; end
 
   describe "#generate_token" do
     it "generates a token using the default method" do
-      FactoryBot.create :access_token
```

Comment:
I think this is placed here by mistake, feel free to comment if it's done on purpose.